main/settings.py

```diff
@@ -34,8 +34,11 @@
     CELERY_REDIS_URL=str,
     DJANGO_STATIC_ROOT=(str, os.path.join(BASE_DIR, "assets/static")),  # Where to store
     DJANGO_STATIC_URL=(str, "/static/"),
+    GIDD_CLIENT_ID=str,
 )
 
+GIDD_CLIENT_ID = env("GIDD_CLIENT_ID")
```

Comment:
This is the ID for both GIDD and IDU. Maybe we can replace this with `IDMC_CLIENT_ID`.